DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py

```diff
@@ -110,14 +135,53 @@
 clusterBin       ['jetCoreRegionalStep'] = cms.int32(500)
 clusterMax       ['jetCoreRegionalStep'] = cms.double(100000)
 
-selectedIterTrackingStep.extend( ['initialStep']  )
-selectedIterTrackingStep.extend( ['lowPtTripletStep']  )
-selectedIterTrackingStep.extend( ['pixelPairStep']  )
-selectedIterTrackingStep.extend( ['detachedTripletStep']  )
-selectedIterTrackingStep.extend( ['mixedTripletStep']  )
-selectedIterTrackingStep.extend( ['pixelLessStep']  )
-selectedIterTrackingStep.extend( ['tobTecStep']  )
-selectedIterTrackingStep.extend( ['jetCoreRegionalStep'] )
-selectedIterTrackingStep.extend( ['muonSeededStepInOut']  )
-selectedIterTrackingStep.extend( ['muonSeededStepOutIn'] )
-#selectedIterTrackingStep.extend( ['muonSeededStepOutInDisplaced'] )
+selectedIterTrackingStep = [
+    'initialStep',
+    'lowPtTripletStep',
+    'pixelPairStep',
+    'detachedTripletStep',
+    'mixedTripletStep',
+    'pixelLessStep',
+    'tobTecStep',
+    'jetCoreRegionalStep',
+    'muonSeededStepInOut',
+    'muonSeededStepOutIn',
+#    'muonSeededStepOutInDisplaced'
+]
+selectedIterTrackingStep_trackingLowPU = [
+    "initialStep",
+    "lowPtTripletStep",
+    "pixelPairStep",
+    "detachedTripletStep",
+    "mixedTripletStep",
+    "pixelLessStep",
+    "tobTecStep",
+    "muonSeededStepInOut",
+    "muonSeededStepOutIn",
+]
+selectedIterTrackingStep_trackingPhase1 = [
+    "initialStep",
+    "highPtTripletStep",
+    "detachedQuadStep",
+#    "detachedTripletStep",
+    "lowPtQuadStep",
+    "lowPtTripletStep",
+    "mixedTripletStep",
+    "pixelLessStep",
+    "tobTecStep",
+    "jetCoreRegionalStep",
+    "muonSeededStepInOut",
+    "muonSeededStepOutIn",
+]
+selectedIterTrackingStep_trackingPhase1PU70 = [
+    "initialStep",
+    "highPtTripletStep",
+    "lowPtQuadStep",
+    "lowPtTripletStep",
+    "detachedQuadStep",
+    "mixedTripletStep",
+    "pixelPairStep",
+    "tobTecStep",
+    "muonSeededStepInOut",
+    "muonSeededStepOutIn",
+]
```

Comment:
This is now the third place where the same information about iterations is copy-pasted (two others being https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/iterativeTk_cff.py and https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Validation/RecoTrack/python/TrackValidation_cff.py). To ease maintenance I'd very much like to specify the information only once (therefore that must be under `RecoTracker/IterativeTracking`). I have already an idea for that and will prototype that.